### PR TITLE
Account for other type of redirect source path

### PR DESCRIPTION
**Pull request by gewarren**
Valid options are source_path and source_path_from_root, which starts with a "/".

@Youssef1313

### CODE REVIEW

**Review by Youssef1313 — APPROVED** [review]
LGTM (assuming it was tested and it works)

**Comment by gewarren** [comment]
> LGTM (assuming it was tested and it works)

How do I test it?

**Comment by Youssef1313** [comment]
A draft PR like https://github.com/dotnet/docs/pull/27365 will use the `source-path`. This works without having to merge because the yml file is using `pull_request`, not `pull_request_target`.

File: actions/docs-verifier/src/RedirectionVerifier/RedirectionsVerifier.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public static class RedirectionsVerifier
                 throw new ArgumentNullException(nameof(writer));
             }
 
-            List<Redirection> foundRedirections = redirections.Where(redirection => redirection.SourcePath == sourcePath).ToList();
+            List<Redirection> foundRedirections = redirections.Where(redirection => redirection.SourcePath == sourcePath || redirection.SourcePathFromRoot == "/" + sourcePath).ToList();
```
Comment (IEvangelist) [inline actions/docs-verifier/src/RedirectionVerifier/RedirectionsVerifier.cs:24]:
```suggestion
            List<Redirection> foundRedirections =
                redirections.Where(
                    redirection =>
                        redirection.SourcePath == sourcePath ||
                        redirection.SourcePathFromRoot == $"/{sourcePath}").ToList();
```

**Review by IEvangelist — APPROVED** [review]
I just have a format nit (for readability and string interpolation). Do we need to read redirection source files from different locations?

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

I like @IEvangelist 's suggestion as well.

**Comment by gewarren** [comment]
> Do we need to read redirection source files from different locations?

I've logged #36 to track that.

**Pull request merged**